### PR TITLE
(COV-805): feat/mapLastUpdatedDate: add last_updated to map legend

### PR DESCRIPTION
**Pull request by ocshawn**
Jira Ticket: [COV-805](https://occ-data.atlassian.net/browse/COV-805)

### New Features
- Add a timestamp of when we synced with JHU to IL and world map legend

### CODE REVIEW

**Comment by ocshawn** [comment]
code can be tested in QA https://qa-covid19.planx-pla.net/

**Review by mfshao — COMMENTED** [review]
1. let's use camel cases for all variable names in portal code
2. the `jsonByLevel` props has defined proptypes as `jhuJsonByLevelLatest`, you should update that definition to reflect the its change

File: src/Covid19Dashboard/index.jsx [hunk 1]
```diff
@@ -415,7 +415,7 @@ Covid19Dashboard.propTypes = {
 Covid19Dashboard.defaultProps = {
   modeledFipsList: [],
   jhuGeojsonLatest: { type: 'FeatureCollection', features: [] },
-  jhuJsonByLevelLatest: { country: {}, state: {}, county: {} },
+  jhuJsonByLevelLatest: { country: {}, state: {}, county: {}, last_updated: '' },
```
Comment (ocshawn) [inline src/Covid19Dashboard/index.jsx:418]: is this what you were referring to by point #2 ?

**Review by mfshao — APPROVED** [review]
👍

**Pull request merged**